src/main/java/org/folio/service/manager/input/InputDataManagerImpl.java

```diff
@@ -255,6 +260,24 @@ private boolean isNotCQL(FileDefinition requestFileDefinition) {
     return !CQL.equals(requestFileDefinition.getUploadFormat());
   }
 
+  private void populateJobExecutionForEmptyFileUploaded(JobExecution jobExecution, FileDefinition fileExportDefinition, JsonObject user) {
```

Comment:
the method looks very similar to the prepareJobForExport(). Maybe. we can move a common part to one place

Reply:
Thank for note, I've added changes regarding your suggestion.